### PR TITLE
[v7r0] Syntax changes to Core package, driven mostly by python-modernize

### PR DESCRIPTION
**Pull request by fstagni**
I used some of the features of python-modernize on the Core package (only). This was mostly a test, and I am quite happy about it. I suggest to run this tool every time a file is modified or created. I mostly looked at https://portingguide.readthedocs.io/en/latest/index.html for info.

While looking, I discovered that some modules that we use will be removed in python3, like `stat`. I did not make changes on this topic in this PR. And we should be careful with urllib.

BEGINRELEASENOTES

*Core
FIX: syntax changes driven mostly by python-modernize

ENDRELEASENOTES

### CODE REVIEW

**Comment by chaen** [comment]
All these changes seem okay to me. They add very little on top of autopep8 however (except the `raw_input` ,  preventing from using protected keywords (`type`), and the `Long` type). 
But why not ! It will probably help us. Shall we add as a unit test that the modified files should remain unchanged if sent through this module ?

File: Core/Utilities/ClassAd/ClassAdLight.py [hunk 1]
```diff
@@ -307,8 +298,7 @@ def getAttributeBool( self, name ):
       return True
     elif value.lower() == "false":
       return False
-    else:
-      return False
+    return False
```
Comment (andresailer) [inline Core/Utilities/ClassAd/ClassAdLight.py]: This whole `if...elif....` could just be
`return value.lower() == "true"`

File: Core/Utilities/Graphs/BarGraph.py [hunk 2]
```diff
@@ -35,7 +31,7 @@ class BarGraph( PlotBase ):
   def __init__(self,data,ax,prefs,*args,**kw):
 
     PlotBase.__init__(self,data,ax,prefs,*args,**kw)
-    if self.prefs.has_key('span'):
+    if 'span' in self.prefs:
```
[the remark this answers is not in the document]
Comment (fstagni) [reply Core/Utilities/Graphs/BarGraph.py]: can't, see the few lines that follow (also true for QualityMapGraph)

**Comment by fstagni** [comment]
Thanks, made the fixes.

**Comment by atsareg** [comment]
We should agree how to proceed with this code modernizing: one commit for running all fixers on per file ? commit per fixer on a whole package ? Making a release for just fixers ? If it will be as chaotic as autopep8 application now, this will be a merging nightmare.

**Pull request merged**